### PR TITLE
chore(upgrade): upgrade golang (1.14.3) and add tools

### PR DESCRIPTION
**Pull request by catsoap**
https://golang.org/doc/devel/release.html#go1.14

I brought back some tools for CI and dev

### CODE REVIEW

File: golang/Dockerfile.j2 [hunk 1]
```diff
@@ -2,6 +2,14 @@ FROM golang:{{BASE_IMAGE_VERSION}}
 LABEL maintainer="Raphaël Benitte <raphael.benitte@ekino.com>"
 
 ARG CI_HELPER_VERSION
+ARG CI_HELPER_VERSION
```
Comment (Bladrak) [inline golang/Dockerfile.j2:5]: Duplicated line

**Comment by catsoap** [comment]
PS: my last push just added gitleaks

File: config.yml [hunk 2]
```diff
@@ -91,13 +91,29 @@ golang:
...
+        - golangci-lint --version
+        - migrate -version
+        - mockgen -version
+        - modd --version
+        - swagger version
+        - testfixtures --version
+
     template_vars:
-      BASE_IMAGE_VERSION: 1.14
+      BASE_IMAGE_VERSION: 1.14.2
```
Comment (mremi) [inline config.yml:116]: https://groups.google.com/forum/#!msg/golang-announce/-9KWN-OUSl0/Jro9NDu8CgAJ :)

Comment (catsoap) [reply config.yml:116]: thanks! waiting for the image release

Comment (catsoap) [reply config.yml]: :heavy_check_mark:  upgraded the image

**Pull request merged**